doc/source/functions/gamma.txt

```diff
@@ -5,16 +5,15 @@ Factorials and factorial-like sums and products are basic tools of combinatorics
 
 A large selection of factorial-like functions is implemented in mpmath. All functions support complex arguments, and arguments may be arbitrarily large. Results are numerical approximations, so to compute *exact* values a high enough precision must be set manually::
 
+    >>> from mpmath import mp, fac
     >>> mp.dps = 15; mp.pretty = True
     >>> fac(100)
     9.33262154439442e+157
-    >>> print int(_)    # most digits are wrong
-    93326215443944150965646704795953882578400970373184098831012889540582227238570431
-    295066113089288327277825849664006524270554535976289719382852181865895959724032
+    >>> print int(_)    # most digits are wrong  # doctest:+SKIP
```

Comment:
Sorry, I was thinking about long() :)  print statement (using print function) should be fixed instead.  Now done.